scalableminds/webknossos
# Add allowsUpload flag to datastores

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- set allowsUpload flag to false via REST API or directly in the database for a datastore
- that datastore should no longer be selectable in dataset upload view

### Issues:
- fixes #4606 

------
